app/src/main/java/com/etoitau/giftessera/domain/DrawSession.kt

```diff
@@ -124,9 +124,9 @@ class DrawSession constructor(private val mainActivity: MainActivity, private va
 
     private fun buildClearAlert(): AlertDialog {
```

Comment:
Minor nitpick: this function could be renamed ("new" instead of "clear")

Reply:
yeah I guess it's clearer if ui naming matches internal better. See next commit